src/In.ProjectEKA.DefaultHip/DataFlow/Collect.cs

```diff
@@ -25,12 +27,14 @@ public Collect(string careContextMapFile)
 
         public async Task<Option<Entries>> CollectData(DataRequest dataRequest)
         {
-            var bundles = new List<Bundle>();
+            var bundles = new Dictionary<string, Bundle>();
             var results = FindPatientData(dataRequest);
-            foreach (var item in results)
+            var careContextReferences = results.Keys.ToList();
+            foreach (var careContextReference in careContextReferences)
             {
-                Log.Information($"Returning file: {item}");
-                bundles.Add(await FileReader.ReadJsonAsync<Bundle>(item));
+                var som = results.GetOrDefault(careContextReference);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'som'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5414977)

Reply:
done